lib/spack/spack/package.py

```diff
@@ -973,6 +973,80 @@ def url_for_version(self, version):
         return spack.url.substitute_version(
             default_url, self.url_version(version))
 
+    def all_urls_for_version(self, version):
+        """Returns all URLs derived from version_urls(), url, urls, and
+        list_url (if it contains a version) in a package in that order.
+
+        version: class Version
+            The version for which a URL is sought.
+        use_list_url: bool
```

Comment:
This parameter does not exist

Reply:
Ah, thanks for that, forgot to remove it in a refactor.